Ta/i like purple butterflies

taranguest opened the pull request:
I think this it is? Not completely sure I have pushed my branch back to the repo? Cool learnings!

lauchness commented on README.md:
```diff
@@ -1 +1 @@
-# learn-react
\ No newline at end of file
+# I __love__ *REACT*
```
No need to change the original file

lauchness commented on README.md.txt:
```diff
@@ -0,0 +1 @@
+#I __love__ *REACT*
```
So this is good, but the file shouldn't be a `.txt`, just a `.md`. Also, put it in a subfolder. Once you're done that, commit your local changes and push them back up here.

Nice work!

lauchness replied:
This file still exists, it should be deleted

lauchness commented:
So, you need to make your changes to this branch, and push them back up here. That way I can review them in this PR. Once they're resolved, we'll merge this PR.

taranguest commented:
I think I made the changes to the actual branch now - removed the .txt from
the file and added a sub folder called TA. Took some playing around and
tried to delete the unnecessary PRs. Is this still attached to the
initial PR?

On Tue, Mar 10, 2020 at 4:31 PM lauchness <notifications@github.com> wrote:

> So, you need to make your changes to this branch, and push them back up
> here. That way I can review them in this PR. Once they're resolved, we'll
> merge this PR.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lauchness/learn-react/pull/1?email_source=notifications&email_token=AOY22VKXDEU4U2KJW2FLR4DRG2ISRA5CNFSM4LFFZOL2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEOM2ORY#issuecomment-597272391>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOY22VMBXJPZLJTLKCWW54LRG2ISRANCNFSM4LFFZOLQ>
> .
>

lauchness commented on README.md:
```diff
@@ -1 +0,0 @@
-# learn-react
```
Looks like you deleted the original README.md, leave this one alone

lauchness replied:
i.e., put it back the way it was so there is no diff in your PR

lauchness replied:
Ok, so the file is back, but you can see here that there's a diff.

The red shows what used to be in the file, and the green shows what's being changed by your PR

lauchness commented:
Ok, we're getting there. Only one thing left, you've made changes to the original README.md (you deleted it). 

You can undo those changes by finding the commit hash that removed the file OR you can just make the file again and make sure it matches what's in master on this repo.
